coral/src/app/router.tsx

```diff
@@ -349,7 +356,118 @@ const routes: Array<RouteObject> = [
   },
 ];
 
-const router = createBrowserRouter(routes, {
+/** `superadminRouteMap` is used as in-between-solution
+ * to be able to filter "routes" and decide which ones
+ * are accessible for role SUPERADMIN. It will be replaced
+ * by a proper routing etc. based on fine-grained permissions.
+ * The `redirect` placeholder (:placeHolder)
+ * has to follow the same pattern as in our
+ * `routes` list to be able to replace it correctly
+ */
+const superadminRouteMap: SuperadminRouteMap = {
+  [Routes.TOPIC_OVERVIEW]: {
+    route: Routes.TOPIC_OVERVIEW,
+    redirect: "/topicOverview?topicname=:topicName",
+  },
+  [Routes.CONNECTOR_OVERVIEW]: {
+    route: Routes.CONNECTOR_OVERVIEW,
+    redirect: "/connectorOverview?connectorName=:connectorName",
+  },
+  [Routes.ENVIRONMENTS]: {
+    route: Routes.ENVIRONMENTS,
+    redirect: "/envs",
+  },
+  [Routes.TEAMS]: {
+    route: Routes.TEAMS,
+    redirect: "/teams",
+  },
+  [Routes.USERS]: {
+    route: Routes.USERS,
+    redirect: "/users",
+  },
+  [Routes.CLUSTERS]: {
+    route: Routes.CLUSTERS,
+    redirect: "/clusters",
+  },
+
+  // These are all routes that don't have an
+  // equivalent in Klaw for SUPERADMIN
+  // and are also unlikely for a superadmin
+  // to click, so redirect goes to not found
+  // could also be dashboard, but not found
+  // gives additional information
+  [Routes.CONNECTOR_REQUEST]: {
```

Comment:
Yes, that's a good point, will add it!

Reply:
Updated! https://github.com/Aiven-Open/klaw/pull/2303/commits/19cd706cae1af5012f97daa705ef83925c985587

I added "(mostly)" because atm some routes _are linked, e.g. the requests and approvals in the navigation at top.